src/eventSchemas/models/schemaCodeLangs.ts

```diff
@@ -9,9 +9,10 @@ import { Set as ImmutableSet } from 'immutable'
 export const JAVA = 'Java 8+'
 export const PYTHON = 'Python 3.6+'
 export const TYPESCRIPT = 'Typescript 3+'
-export type SchemaCodeLangs = 'Java 8+' | 'Python 3.6+' | 'Typescript 3+'
+export const GO = 'Go 1+'
```

Comment:
We should probably ask the Schemas team to make sure this is all supported. This is not part of our core Lambda/SAM flow.

Reply:
Removed that code for now.